app/templates/home.hbs

```diff
@@ -50,6 +50,13 @@
             </div>
         </div>
     </div>
+    {{#if isOpen}}
+         <div class="alert alert-warning alert-message">
+            <strong>Warning!</strong> This OSF project is <strong>not public </strong> and can not be viewed.<br /> Please consider making this project public!<br />
```

Comment:
Can you rework the language to be more clear like ```the pages you create will not be visible because the project is not public```

Also if I remember correctly didn't we say if the project is not public they can still use non project related layers?

Reply:
yes we can reword this, and I thought we came to the conclusion that any project that is private should not be visible to a non-contributor